mods/ra/weapons/explosions.yaml

```diff
@@ -126,8 +126,8 @@ SmallBuildingExplode:
 CivBuildingExplosion:
 	Inherits: SmallBuildingExplode
 	Warhead@1Dam: SpreadDamage # Used to panic civilians which are emitted from a killed CivBuilding
-		Spread: 64
-		Damage: 1
+		Spread: 128
```

Comment:
IMO something like the following might be better here:
```yaml
Falloff: 100, 100
Range: 0, 2c0 # Replace 2c0 with the actual maximum we want here
Damage: 1
```

Reply:
Agreed. Damage 100 would be enough to damage nearby infantry and vehicles, I doubt we want that.

Reply:
It didn't appear to create any damage to surrounding units, so I assumed it'd be fine, but I'll make the changes. -- Someone will need to squash my commits though :(

Reply:
> Someone will need to squash my commits though :(

No problem.

Reply:
I'm now thinking a separate explosion for CivBuilding might make the most sense as civilians spawn from the center, and buildings use footprint explosions.
```
[rules/defaults.yaml]
^CivBuilding:
	Explodes:
		Weapon: SmallBuildingExplode
	Explodes@CIVPANIC:
		Weapon: CivBuildingExplosion
		#Type: CenterPosition (default value)


[weapons/explosions.yaml]
CivBuildingExplosion:
	-Inherits: SmallBuildingExplode
	Warhead@1Dam: SpreadDamage
		-Spread:
		Falloff: 100, 100
		Range: 0, 128
		Damage: 1
		Delay: 1
````

In which case should I rename CivBuildingExplosion to CivPanicExplosion? -- since it'll no longer be inheriting SmallBuildingExplosion.

Reply:
That Inherits line si full bonkers. Go for it otherwise.

Reply:
My intention was simply to indicate its removal. (which I should have done with a comment instead of "-").